main.nf

```diff
@@ -795,27 +847,29 @@ process run_percolator {
 
     script:
     if (params.description_correct_features > 0 && params.klammer){
+
     """
-    PercolatorAdapter -in ${id_file_psm} \\
-                       -out ${id_file_psm.baseName}_perc.idXML \\
+    OMP_NUM_THREADS=${task.cpus} PercolatorAdapter -in ${id_file_psm} \\
```

Comment:
This sets an ENV variable right?
Maybe add a new line before calling Percolator to separate this more clearly?

Reply:
solved